cmd/gaia/init/testnet.go

```diff
@@ -85,7 +85,7 @@ Example:
 	)
 	cmd.Flags().String(
 		server.FlagMinGasPrices, fmt.Sprintf("0.000006%s", sdk.DefaultBondDenom),
-		"Minimum gas prices to accept for transactions; All fees in a tx must meet this minimum (e.g. 0.01photino,0.001stake)",
+		"Minimum gas prices to accept for transactions; All fees in a tx must meet this minimum (e.g. 0.01photino,0.001atom)",
```

Comment:
testnets _do not_ use atoms but actually use an imaginary token named stake (aka this PR change should be reversed)